cmd/gokr-packer/packer.go

```diff
@@ -106,6 +107,12 @@ You can also create your own certificate-key-pair (e.g. by using https://github.
 	testboot = flag.Bool("testboot",
 		false,
 		"Trigger a testboot instead of switching to the new root partition directly")
+
+	deviceType = flag.String("device_type",
+		"",
+		`Device type identifier (defined in github.com/gokrazy/internal/deviceconfig) used for applying device-specific modifications to gokrazy.
+e.g. -device_type="Hardkernel Odroid HC1" to apply MBR changes and device-specific bootloader files for Odroid XU4/HC1/HC2.
```

Comment:
Thanks for adding the example.

`-device_type="Hardkernel Odroid HC1"` is rather long to type in. 

On https://gokrazy.org/platforms/#continuous-integration-ci, we have architecture target names that are shorter. What do you think about `odroid` as name? Or `odroidhc1` if it needs to be more specific?

Reply:
Hmmm, I'm using the value returned by `gokrazy.Model` to as the key in `DeviceConfigs` in https://github.com/gokrazy/internal/blob/main/deviceconfig/config.go#L30. If we want a shorter name, I can add a `Slug` field to `DeviceConfig` and have something like `GetDeviceConfigBySlug(slug string)` in the internal package.

Let me know what you think.

Reply:
Yep, adding the slug field sounds good! The model name is good to display, but a slug will be more convenient to use, I think.

Reply:
Created a pull request for deviceconfig change in https://github.com/gokrazy/internal/pull/10

Reply:
Thanks, now merged! Can you update go.mod accordingly please?

Reply:
Done